python/pp_ser.py

```diff
@@ -324,13 +324,11 @@ def __ser_register(self, args):
 
         # data type
         datatypes = dict(integer=["'int'", 'ppser_intlength'], real=['ppser_realtype', 'ppser_reallength'])
+
+        if dirs[1] not in datatypes:
+            self.__exit_error(directive = args[0], msg = 'Data type ' + dirs[1] + ' is not recognized. Valid type are "integer" and "real"')
```

Comment:
I propose to have the following message:

    msg = 'Data type "{0}" not understood. Valid types are: {1}'.format(dirs[1], ', '.join('"' + d + '"' for d in datatypes.keys()))